src/common_coin.rs

```diff
@@ -1,4 +1,26 @@
-//! Common coin from a given set of keys based on a `pairing` threshold signature scheme.
+//! # A Cryptographic Common Coin
+//!
+//! The Common Coin produces a pseudorandom binary value that the correct nodes agree on, and that\
+//! cannot be known beforehand.
+//!
+//! Every Common Coin instance has a _nonce_ that determines the value, without giving it away: It
+//! is not feasible to compute the output from the nonce alone, and for any "natural" sequence of
+//! nonces (that can be feasibly computed), the output is uniformly distributed.
+// TODO: Bold claims! How much can we actually prove about that?
```

Comment:
A proof would require determining wether the expected number of zeros or ones in a signature is 50%. In the absence of such proof, can we refer to `tests/common_coin.rs` for that? If you flip the common coin like a 1000 times, you should get pretty much 50% of each of `true` and `false`. There is no test with 1000 flips because of time constraints.

Reply:
Yes, it's just that that's not strictly a proof. What I'm kind of claiming here is that there is no way to produce an efficiently computable sequence of nonces whose output isn't uniformly distributed. All we need is just that sequences like `"nonce 1", "nonce 2", "nonce 3", ...` will work fine. Not sure what's the best formulation.

Reply:
I changed it to a much simpler formulation and don't mention sequences at all. I think the meaning is still pretty clear this way.